### PR TITLE
Function to obtain NSB tuning parameters

### PR DESCRIPTION
**Pull request by moralejo**
This function reads a real data and an MC file and returns the parameters needed to tune the MC NSB to that of the data.

### CODE REVIEW

**Review by vuillaut — DISMISSED** [review]

File: lstchain/image/modifier.py [hunk 1]
```diff
@@ -105,10 +121,235 @@ def random_psf_smearer(image, fraction, indices, indptr):
...
+
+    # Just in case, makes sure we just add noise if the MC noise is smaller
+    # than the real data's:
+    extra_noise_in_bright_pixels = max(0., extra_noise_in_bright_pixels)
+
+    bias = mode_data - mode_mc
+    extra_bias_in_dim_pixels = max(bias, 0)
+
+    # differences of values to peak charge:
+    dq = data_ped_charges[all_good].flatten() - mode_data
```
Comment (rlopezcoto) [inline lstchain/image/modifier.py:344]: what is `dq`? `charge_difference`? maybe explicitly naming it?

File: lstchain/image/modifier.py [hunk 2]
```diff
@@ -105,10 +121,235 @@ def random_psf_smearer(image, fraction, indices, indptr):
...
+    dqmc = mc_ped_charges_biased - mode_mc
```
Comment (rlopezcoto) [inline lstchain/image/modifier.py:345]: same as above

File: lstchain/scripts/lstchain_tune_nsb.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
+"""
+Usage:
+
+$> python lstchain_tune_nsb.py
+--config  config_file.json     (must be the one used in the DL1 production)
+--input-mc  simtel_file.simtel.gz    simulation simtel file
+--input-data_dl1 dl1_data.h5         real data DL1 file
```
Comment (rlopezcoto) [inline lstchain/scripts/lstchain_tune_nsb.py:7]: `input-data_dl1` -> `input-data`

File: lstchain/image/modifier.py (hunk 1 above)
Comment (moralejo) [reply lstchain/image/modifier.py:344]: Would make the later expressions clumsy. And it is clearly explained, right? Differences of values to peak charge (dq stands for delta_charge). Also, they are used very close to their initialization so it is quite clear I think.

File: lstchain/image/modifier.py [hunk 4]
```diff
@@ -105,10 +121,235 @@ def random_psf_smearer(image, fraction, indices, indptr):
...
+    # maximum distance (in pe) from peak, to avoid strong impact of outliers:
+    maxq = 10
+    # calculate widening of the noise bump:
+    added_noise = (np.sum(dq[dq<maxq]**2) / len(dq[dq<maxq]) -
+                   np.sum(dqmc[dqmc<maxq]**2) / len(dqmc[dqmc < maxq]))
+    added_noise = (max(0, added_noise)) ** 0.5
```
[the remark this answers is not in the document]
Comment (moralejo) [reply lstchain/image/modifier.py:351]: ok, I did some changes. Anyway, among the pep 8 recommendations for placing spaces I see "Use your own judgment" :)

**Review by rlopezcoto — APPROVED** [review]

**Pull request merged**